app/assets/config/manifest.js

```diff
@@ -6,3 +6,4 @@
 //= link s3_direct_upload.js
 // = link administrate/application.css
 // = link administrate/application.js
+// = link minimal.css
```

Comment:
I'm really not sure about this thing here... More context [here](https://dev-internal.slack.com/archives/C7GE84DEJ/p1583324184050100).

Reply:
I have to look into this as well. Is this related to the recent upgrade of Sprockets @rhymes?

Reply:
Yes, I'm not 100% why it's required since we already include the entire `stylesheets` directory a few lines earlier:

```
//= link_directory ../stylesheets .css
```

but it might be an issue of ordering, sorry I don't have more details yet. Let's keep it it like this for now?